pocket_snowflake/views/pocket_hits_r4_w_avg_click_counts.view.lkml

```diff
@@ -0,0 +1,38 @@
+view: pocket_hits_r4_w_avg_click_counts {
+  sql_table_name: "DBT_SHARED_WITH_MOZILLA"."POCKET_HITS_R4W_AVG_CLICK_COUNTS"
+    ;;
+
+  dimension: click_count {
+    type: number
+    sql: ${TABLE}."CLICK_COUNT" ;;
+  }
+
+  dimension: click_count_r4_w {
+    type: number
+    sql: ${TABLE}."CLICK_COUNT_R4W" ;;
+  }
+
+  dimension_group: happened_week {
```

Comment:
In the other view files, we changed the `happened_week` name to `Period` for easier readability.We should keep it consistent.

Reply:
I've changed the name of the time frame to period. I added a second dimension group to filter the date to be this year and up to today. The second date dimension group is hidden, but you can see how it is filtered